2022-finalproject/BlanchardLj/git.md

```diff
@@ -0,0 +1,23 @@
+1. 若你已经修改了部分文件、并且将其中的一部分加入了暂存区，应该如何回退这些修改，恢复到修改前最后一次提交的状态？给出至少两种不同的方式
+   
+   ![](./1-1.PNG)
+
+   ![](./1-2.PNG)
+
+   ![](./1-3.PNG)

```

Comment:
You just modified the files in the index, but the task required you to return the working area to the condition of the last commit,.

Reply:
OK, I have changed it. Please check it again. @irvingskr